input/input_scan_fullsim.txt

```diff
@@ -0,0 +1,18 @@
+#sections: global options in OPTION, selections in SELECTION, variations in VARIATION, datasets in SET
```

Comment:
why can't you just combine this file with `input/input_scan.txt`?

Reply:
Combined and `input/input_scan.txt` removed. I also squashed the commits so that this file never enters the history.